### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by MegiJ**
Dodałam test shouldThrowExceptionWhenOnePartOfNameIsNull sprawdzający czy rzuci wyjątek jeśli imię lub nazwisko będzie null. Dodatkowo sprawdzana jest treść wyjątku.

### CODE REVIEW

File: backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+        //when
+        boolean result = campOrganisation.addStudent("Daniel", "Gie");
+
+        //then
+        Assertions.assertFalse(result);
+    }
+
+    @Test
+    public void shouldThrowExceptionWhenOnePartOfNameIsNull() {
```
Comment (PaniOdProgramowania) [inline backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java:88]: good test! it's name `nameIsNull` though seems not equal to what the params are in line 93 - `("Antoni", "");` (empty String is passed even though the test name suggested `null`). Please unify it :)

File: backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java [hunk 2]
```diff
@@ -87,13 +89,15 @@ public void shouldThrowExceptionWhenOnePartOfNameIsNull() {
         //given
         //when
         //then
-        Assertions.assertThrows(IllegalArgumentException.class, () -> {
+        Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
```
Comment (PaniOdProgramowania) [inline backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java:92]: `Throwable` is the highest class in the hierarchy here. We want to test if it's exactly `IllegalArgumentException`. We are checking it in `assertThrows` but we can add a double check by expecting here not `Throwable` but `IllegalArgumentException` :)

**Review by PaniOdProgramowania — COMMENTED** [review]
great work!

File: backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java (hunk 1 above)
Comment (MegiJ) [reply backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java:88]: Ok, but I thought that empty string is just null? I should write instead "null" - "emptyString" ? shouldThrowExceptionWhenOnePartOfNameIsEmptyString?

**Review by PaniOdProgramowania — APPROVED** [review]
🎉

**Pull request merged**